angular-primeng-app/src/app/pipes/truncate.pipe.ts

```diff
@@ -0,0 +1,18 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
```

Comment:
@DongreJaipal please remove this file the - `angular-primeng-app/src/app/pipes/truncate.pipe.ts` 

the two files for the pipe should not be included the rest is ok

Reply:
okay @cleancodecraft